### PR TITLE
feat(www): 🌐 introduce new i18n languages support

### PR DESCRIPTION
**Pull request by jyyi1**
In this PR, I introduced 9 more language support to the Client, including:

* bs
* en-GB
* es
* et
* hy
* ka
* mk
* pt-PT
* sq

In order to display the new languages in the UI, I updated the language list as well. The language names come from the official [Google Account language settings page](https://myaccount.google.com/language).

I also updated the translations for the following languages:

* fa
* my
* pt-BR

Related PR: https://github.com/Jigsaw-Code/outline-server/pull/1100.

### CODE REVIEW

**Comment by daniellacosse** [comment]
Probably worth adding `messages` to the list of things we ignore in the pr size checker.

File: src/www/messages/iw.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
```
Comment (fortuna) [inline src/www/messages/iw.json:1]: `iw` is deprecated and shouldn't be used:
https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes#he

Don't we already have `hw` for Hebrew?

File: src/www/ui_components/app-root.js [hunk 2]
```diff
@@ -408,77 +408,87 @@ export class AppRoot extends mixinBehaviors([AppLocalizeBehavior], PolymerElemen
...
-          "sr-Latn": {id: "sr-Latn", name: "Srpski", dir: "ltr"},
-          sv: {id: "sv", name: "Svenska", dir: "ltr"},
-          th: {id: "th", name: "ไทย", dir: "ltr"},
-          tr: {id: "tr", name: "Türkçe", dir: "ltr"},
-          uk: {id: "uk", name: "Українська", dir: "ltr"},
-          ur: {id: "ur", name: "اردو", dir: "rtl"},
-          vi: {id: "vi", name: "Việtnam", dir: "ltr"},
-          "zh-CN": {id: "zh-CN", name: "简体中文", dir: "ltr"},
-          "zh-TW": {id: "zh-TW", name: "繁體中文", dir: "ltr"},
+          am: {id: 'am', name: '‪አማርኛ‬', dir: 'ltr'},
```
Comment (fortuna) [inline src/www/ui_components/app-root.js:430]: I don't understand why you need the hidden characters. Unicode has rules to change direction already, and it's not like this is inserted in text.

Do you have an example of where this is broken?

Comment (jyyi1) [reply src/www/ui_components/app-root.js:430]: I don't aware of any broken languages for now. But these characters are controlling the direction of a mixed environment, for example, the `C++` string mentioned in https://en.wikipedia.org/wiki/Left-to-right_mark

File: src/www/messages/iw.json (hunk 1 above)
Comment (jyyi1) [reply src/www/messages/iw.json]: Right, `he` and `iw` are identical.

File: src/www/ui_components/app-root.js (hunk 2 above)
Comment (jyyi1) [reply src/www/ui_components/app-root.js]: As discussed offline, I removed these bidi characters.

**Review by fortuna — APPROVED** [review]

**Pull request merged**